src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -229,6 +229,11 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
 
                 project.RawRuntimeOptions = runtimeOptions.ToString();
             }
+            else if (File.Exists(Path.Combine(projectPath, $"{projectName}.runtimeconfig.json")))
```

Comment:
@livarcocc There is a [bug](https://github.com/dotnet/cli/issues/2485) that specifies that we need to look at this file so this is fixing that bug since we do not look for it today. I'm not sure what the full story is, just fixing an RC2 bug :)

Merging would lead to lots of problems (such as two conflicting values) so we should avoid that.